GIT_AND_GITHUB.md

```diff
@@ -42,26 +42,39 @@ refactor/make-it-awesome
 
 Use `--no-ff` when you merge to `master`.
 
-### Merging
-
 It's okay to just push to `master` if it's a quick fix and you really need
 it out the door. Otherwise, make it a pull request.
 
+### Pull requests
+
 Pull request descriptions should be concise and well written. The merger should
 be able to copy this description straight into the release notes instead of
 figuring out what changed or was fixed.
 
-#### Who merges the pull request?
+Besides a clear description more information could be helpful, for example:
```

Comment:
`Besides a clear description, more information could be helpful, for example:` missed a comma here.